pkg/cmd/server/server.go

```diff
@@ -67,6 +67,7 @@ func (o *options) addFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&o.serverConfig.TZ, "tz", o.serverConfig.TZ, "Specify time zone of TiCDC cluster")
 	cmd.Flags().Int64Var(&o.serverConfig.GcTTL, "gc-ttl", o.serverConfig.GcTTL, "CDC GC safepoint TTL duration, specified in seconds")
+	cmd.Flags().Int64Var(&o.serverConfig.MaxElapsedTime, "max-elapsed-time", o.serverConfig.MaxElapsedTime, "CDC MaxElapsedTime duration, specified in minutes")
```

Comment:
thanks for your review,i've tolk to  @leoppro that the pr will not be passed.
i am trying to pr for cdc when learning cdc.

i have a propose that can we add help wanted label for tiflow repo like tidb repo.
so that people can good participate in tiflow for newer.